pyproject.toml

```diff
@@ -10,6 +10,8 @@ name = "reboost"
 description = "New LEGEND Monte-Carlo simulation post-processing"
 authors = [
     { name = "Manuel Huber", email = "info@manuelhu.de" },
+    { name = "Toby Dixon", email = "toby.dixon.23@ucl.ac.uk" },
+    { name = "Luigi Pertoldi", email = "gipert@pm.me" },
```

Comment:
@gipert just fyi: this is not displayed on pypi... (it's a pypi problem, the metdata in the built wheel is correct)

see (just to name some): https://github.com/pypi/warehouse/issues/12877, https://github.com/pypi/warehouse/issues/9400